packages/mongo/collection_tests.js

```diff
@@ -89,7 +89,13 @@ Tinytest.add('collection - call native find with sort function',
     if (Meteor.isServer) {
       test.throws(
         function () {
-          nativeCollection.find({}, {sort: function(){}}).map(function (doc) { return doc.a; })
+          nativeCollection
+            .find({}, {
+              sort: function () {},
```

Comment:
Though I'm not going to request changes, in the future you should know you can write methods like this using method syntax:
```js
nativeCollection
  .find({}, {
    sort() {},
  })
```